moto/rds2/models.py

```diff
@@ -832,12 +838,6 @@ def modify_db_parameter_group(self, db_parameter_group_name, db_parameter_group_
 
         return db_parameter_group
 
-    def delete_db_parameter_group(self, db_parameter_group_name):
```

Comment:
this exact same method was defined twice in this class (copy paste error I guess)